src/api.ts

```diff
@@ -0,0 +1,65 @@
+import axios from 'axios';
+const isDev = process.env.REACT_APP_DEV_MODE === 'true';
```

Comment:
```suggestion

const isDev = process.env.REACT_APP_DEV_MODE === 'true';
```
Can we add a line space here